po/ru.po

```diff
@@ -0,0 +1,355 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR Jos Lemmens <jos@jlemmens.nl>
+# This file is distributed under the same license as the ebook-speaker package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: ebook-speaker 6.2\n"
+"Report-Msgid-Bugs-To: https://github.com/book-readers/ebook-speaker/issues\n"
+"POT-Creation-Date: 2021-08-21 13:35+0200\n"
+"PO-Revision-Date: 2022-02-15 13:04+0300\n"
+"Last-Translator: Vsevolod Popov\n"
```

Comment:
I should have been more precise: we need the name *and* the address :) just like Language-Team